lib/chai/core/assertions.js

```diff
@@ -215,6 +215,16 @@ module.exports = function (chai, _) {
   Assertion.addChainableMethod('contains', include, includeChainingBehavior);
   Assertion.addChainableMethod('includes', include, includeChainingBehavior);
 
+  // TODO: Add jsdoc
+
+  function assertFail (msg) {
+    if (msg) flag(this, 'message', msg);
+    var obj = flag(this, 'object');
+    throw new chai.AssertionError(msg, {}, obj);
+  }
+
+  Assertion.addMethod('fail', assertFail);
+
```

Comment:
@keithamus Can you advise on how to add a `fail` method directly to `expect`? I can't find concrete examples and it seems somewhat impossible because `expect` is a function, unless I'm misunderstanding the architecture.